lib/stargate/producer.ex

```diff
@@ -43,7 +43,7 @@ defmodule Stargate.Producer do
               optional(:replicationClusters) => [String.t()]
             }
 
-  @doc """
+  @typedoc """
```

Comment:
Did you intend to change this particular attribute? Since this annotates the `produce//2` function below this should be a `@doc`; `@typedoc` is used to annotate custom types.